tests/app/test_email_limit_utils.py

```diff
@@ -30,17 +31,9 @@ def test_fetch_todays_requested_email_count(self, client, mocker, sample_service
     def test_increment_todays_requested_email_count(self, mocker, sample_service, redis_value, db_value, increment_by):
         cache_key = email_daily_count_cache_key(sample_service.id)
         mocker.patch("app.redis_store.get", lambda x: redis_value if x == cache_key else None)
-        mocked_set = mocker.patch("app.redis_store.set")
         mocked_incrby = mocker.patch("app.redis_store.incrby")
         mocker.patch("app.email_limit_utils.fetch_todays_email_count", return_value=db_value)
 
         increment_todays_email_count(sample_service.id, increment_by)
 
-        assert mocked_incrby.called_once_with(cache_key, increment_by)
-        if redis_value is None:
-            assert mocked_set.called_once_with(
-                cache_key,
-                db_value,
-            )
-        else:
-            mocked_set.assert_not_called()
```

Comment:
We're mocking the code that involves the set (`fetch_todays_email_count()`) so the set isn't actually called. That's  ok since this code is tested above, so it's cleaner to mock it out in this test (we just can't test the `set` though)